Automattic/wp-calypso
# Add new newsletter intro content

#### Proposed Changes
| Before  | After |
| ------------- | ------------- |
| <img width="723" alt="Screenshot 2022-10-24 at 16 54 36" src="https://user-images.githubusercontent.com/7000684/197557381-d6ac62a9-54f2-429d-b119-9e6c7e861aec.png"> |  <img width="1074" alt="Screenshot 2022-10-24 at 16 53 53" src="https://user-images.githubusercontent.com/7000684/197557468-f69695f8-a6b4-4afa-92af-09e532cdb0f2.png"> |

- We need to update the newsletter intro page according to the new design
- We provide a mechanism to display the jetpack logo in the header
- In case the user locale is English or we have the new translations ready we display the new newsletter intro page.
 
#### Testing Instructions
- To test the new intro page visit the newsletter intro page with english locale. Test with different viewport sizes and compare the designs
- Check the link in bio flow for possible regression
- Check the newsletter flow with locale different than english for possible regressions

Related to #69387
